debian/control

```diff
@@ -77,3 +77,18 @@ Multi-Arch: same
 Depends: nnstreamer, nntrainer, ${shlibs:Depends}, ${misc:Depends}
 Description: Development package for nntrainer tensor filter
  This is a developement package of nntrainer's tensor filter.
+
+Package: nnstreamer-nntrainer-trainer
+Architecture: any
+Multi-Arch: same
+Depends: nnstreamer, nntrainer, ${shlibs:Depends}, ${misc:Depends}
+Description: NNStreamer tensor trainer for nntrainer
+ This is a NNStreamer tensor trainer for nntrainer to support training.
+
+Package: nnstreamer-nntrainer-trainer-dev
```

Comment:
Do we need separate package? How about using same package in nnstreamer-nntrainer ?

Reply:
Here is @myungjoo 's comment:
Since tensor_trainer is unstable,
It is better to separate the build options so that tensor_trainer only can be removed when nntrainer is released.
How about you?

Reply:
As agreed with each other,
When the tensor_trainer sub-plugin is stabilized, we will merge it with the tensor_filter sub-plugin package.

Reply:
Could you leave the comment as an TODO in code?

Reply:
okay,